web/ui/react-app/src/pages/graph/GraphControls.tsx

```diff
@@ -147,6 +148,18 @@ class GraphControls extends Component<GraphControlsProps> {
             <FontAwesomeIcon icon={faChartArea} fixedWidth />
           </Button>
         </ButtonGroup>
+
+        <ButtonGroup className="show-exemplars" size="sm">
+          {this.props.showExemplars ? (
+            <Button title="Hide exemplars" onClick={() => this.props.onChangeShowExemplars(false)} active={true}>
```

Comment:
So I couldn't find this button at all without looking at the code to check where it went :) I like moving it under the graph options, but currently it looks like an unchecked checkbox, and when clicking it it looks like it can't be checked (the background color change is too subtle). There's probably no good icon that makes the idea of exemplars clear enough, so we probably can't do better than just having a `Show exemplars` / `Hide exemplars` as text directly in the button. That, or make it an actual checkbox with a text label `Show exemplars`, but then I'm not sure if/how we'd want to style the checkbox to fit in visually with the other buttons in that bar. Probably just putting Show/Hide text into the button itself is fine for now.

Reply:
Yeah, I agree with you on this one.